TAF_EasyRest/TestFramework/PageComponents/Owner/ChangeItemMenuPageComponent.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace TestFramework.PageComponents.Owner
+{
+    public class ChangeItemMenuPageComponent
+    {
+        private IWebDriver driver;
```

Comment:
I suppose it'd be better to add property here private "IWebDriver driver { get; }"